games-emulation/pcsx2/pcsx2-1.3.1.ebuild

```diff
@@ -0,0 +1,123 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+PLOCALES="ar_SA ca_ES cs_CZ de_DE es_ES fi_FI fr_FR hr_HR hu_HU id_ID it_IT ja_JP ko_KR ms_MY nb_NO pl_PL pt_BR ru_RU sv_SE th_TH tr_TR zh_CN zh_TW"
+
+inherit cmake-utils l10n multilib toolchain-funcs wxwidgets
+
+DESCRIPTION="A PlayStation 2 emulator"
+HOMEPAGE="http://www.pcsx2.net"
+SRC_URI="https://github.com/PCSX2/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+IUSE="+wxwidgets3"
+
+RDEPEND="
+	app-arch/bzip2[abi_x86_32(-)]
+	dev-libs/libaio[abi_x86_32(-)]
+	media-libs/alsa-lib[abi_x86_32(-)]
+	media-libs/libsdl[abi_x86_32(-),joystick,sound]
+	media-libs/portaudio[abi_x86_32(-)]
+	media-libs/libsoundtouch[abi_x86_32(-)]
+	virtual/opengl[abi_x86_32(-)]
+	>=sys-libs/zlib-1.2.4[abi_x86_32(-)]
+	virtual/jpeg:62[abi_x86_32(-)]
+	x11-libs/gtk+:2[abi_x86_32(-)]
+	x11-libs/libICE[abi_x86_32(-)]
+	x11-libs/libX11[abi_x86_32(-)]
+	x11-libs/libXext[abi_x86_32(-)]
+
+	!wxwidgets3? ( x11-libs/wxGTK:2.8[abi_x86_32(-),X] )
+	wxwidgets3? ( x11-libs/wxGTK:3.0[abi_x86_32(-),X] )
+"
+# Ensure no incompatible headers from eselect-opengl are installed, bug #510730
+DEPEND="${RDEPEND}
+	>=app-eselect/eselect-opengl-1.3.1
+	>=dev-cpp/sparsehash-1.5
+"
+
+PATCHES=(
+	"${FILESDIR}"/"${P}-egl-optional.patch"
+	"${FILESDIR}"/"${P}-packaging.patch"
+	"${FILESDIR}"/"${P}-cflags.patch"
+
+)
+
+# Upstream issue: https://github.com/PCSX2/pcsx2/issues/417
+QA_TEXTRELS="usr/lib32/pcsx2/*"
+
+clean_locale() {
+	rm -R "${S}"/locales/"${1}" || die
+}
+
+pkg_setup() {
+	if [[ ${MERGE_TYPE} != binary ]] && [[ $(tc-getCC) =~ gcc ]] && [[ $(gcc-version) < 4.7 ]] ; then
```

Comment:
I would advise against versionator.eclass since it is not reliable. Second try:
```sh
if [[ ${MERGE_TYPE} != binary && $(tc-getCC) == *gcc* ]] ; then
    if [[ $(gcc-major-version) -lt 4 || $(gcc-major-version) == 4 && $(gcc-minor-version) -lt 7 ]] ; then
        eerror "${PN} does not compile with gcc less than 4.7"
        die "${PN} does not compile with gcc less than 4.7"
    fi
fi
```